x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_kpis/alerts_histogram_panel/index.tsx

```diff
@@ -203,24 +214,10 @@ export const AlertsHistogramPanel = memo<AlertsHistogramPanelProps>(
       [onlyField, title]
     );
 
-    const { responses, loading } = useVisualizationResponse({
-      visualizationId,
-    });
-    const embeddedDataLoaded = !loading && !isEmpty(responses);
-    const aggregationBucketsCount = useMemo(
-      () =>
-        loading
-          ? 0
-          : sumBy(responses, (responseItem) =>
-              sumBy(Object.values(responseItem.aggregations ?? {}), 'buckets.length')
-            ),
-      [loading, responses]
-    );
-
-    const embeddedDataAvailable = !!aggregationBucketsCount;
+    const embeddedDataAvailable = totalTableCount != null && totalTableCount > 0;
 
     const subtitle = showHistogram
-      ? createEmbeddedDataSubtitle(embeddedDataLoaded, embeddedDataAvailable, totalAlerts)
+      ? createEmbeddedDataSubtitle(embeddedDataAvailable, totalAlerts)
```

Comment:
Previously we were checking if the embeddable data has been loaded or not but now we are not checking the same. Has something changed on embeddable side?